speech/cloud-client/pom.xml

```diff
@@ -33,25 +33,12 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
   </properties>
 
-  <!-- FIXME(lesv) - temp to fix an issue w/ GA Datastore -->
-<!--
-  <dependencyManagement>
-    <dependencies>
-      <dependency>
-          <groupId>io.grpc</groupId>
-          <artifactId>grpc-core</artifactId>
-          <version>1.2.0</version>
-      </dependency>
-    </dependencies>
-  </dependencyManagement>
- -->
-
   <dependencies>
     <!-- [START dependencies] -->
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud</artifactId>
```

Comment:
Why do we include `google-cloud` (all client libs) instead of `google-cloud-speech` ?